### ISSUE TITLE
Suggestion only: Colour interface vectors should appear for more colour variables

### ISSUE THREAD

**Issue by Const1702** [MattFiler/OpenCAGE#163]
There is a new colour picker shows for "colour" vector parameters as well as "COLOUR_TINT" and "DEPTH_INTERSECT_COLOUR_VALUE" for FogSpheres (probably even more). What would be better is if the colour picker is shown for vector parameters like "lgt_all_col", "lgt_col", "lgt_emissive_col" (for VDUs), which are are mostly used for light composites (not LightReferences, ex: small_floor_light, RotateStripLight_Head, Small_Strip...). As you can see "lgt_col" it is shown with the old vector interface for a RotateStripLight_Head composite:
![image](https://user-images.githubusercontent.com/116276106/224341889-d5e36eb6-ba42-4cc4-b478-aae511b665cd.png)

Now since there are more rare colour parameters I would suggest all vector parameters should have two different view modes. The regular one and the colour picker one. Known colour vectors will have the colour picker view mode selected by default, but there will be a button "Switch to default vector view" and on the default vector view the button will say "Switch to colour picker view" (basically toggling between these two vector interfaces). This switching will exist for all vector parameters.

**Comment by MattFiler** [comment]
Good catch! Annoyingly, Cathode doesn't differentiate between regular vectors and colours, so I've had to list them all manually - missed those ones!

I like the suggestion of swapping between inputs manually. Will have a think about that 👍
